### PR TITLE
chore: update/fix github actions versions

### PR DESCRIPTION
**Pull request by melwynfurtado**
Thanks for this Github Action!

Can we update the two actions used within to the latest version please?

Fix the version for `mikefarah/yq@v4.44.2` so it does not pull whatever goes into master.
And use latest version for `tj-actions/changed-files@v44`

Many thanks once again 🙏🏼 

no qa required

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -15,15 +15,15 @@ inputs:
     required: false
     default: https://axe-linter.deque.com
 runs:
-  using: 'composite'
+  using: "composite"
   steps:
     - uses: actions/checkout@v2
```
Comment (michael-siek) [inline action.yml:20]:
```suggestion
    - uses: actions/checkout@v4
```

File: action.yml [hunk 2]
```diff
@@ -15,15 +15,15 @@ inputs:
...
-    - uses: mikefarah/yq@master
+    - uses: mikefarah/yq@v4.44.2
```
Comment (michael-siek) [inline action.yml:21]:
```suggestion
    - uses: mikefarah/yq@f15500b20a1c991c8729870ba60a4dc3524b6a94 # Tag=v4
```

File: action.yml [hunk 3]
```diff
@@ -15,15 +15,15 @@ inputs:
...
     - name: Get changed files
       id: changed_files
       env:
         GITHUB_TOKEN: ${{ inputs.github_token }}
-      uses: tj-actions/changed-files@v18.7
+      uses: tj-actions/changed-files@v44
```
Comment (michael-siek) [inline action.yml:26]:
```suggestion
      uses: tj-actions/changed-files@6b2903bdce6310cfbddd87c418f253cf29b2dec9 # Tag=v44
```

**Review by michael-siek — CHANGES_REQUESTED** [review]
Hi @melwynfurtado do you mind updating to use pinned version of the GHA deps

**Comment by melwynfurtado** [comment]
@michael-siek thanks for looking into this. Updated to pinned versions.

**Review by michael-siek — APPROVED** [review]

**Comment by michael-siek** [comment]
@melwynfurtado Thank you for the contribution!
- [x] reviewed for security

**Comment by melwyn-furtado** [comment]
@michael-siek how will the versioning work? will this commit tagged as `v1` ?

**Comment by michael-siek** [comment]
Hey @melwyn-furtado I will speak with the team and keep you updated.

**Comment by stephenmathieson** [comment]
@michael-siek what did the team say regarding versioning?

**Pull request merged**